genesis/src/main.rs

```diff
@@ -110,6 +110,7 @@ fn main() -> Result<(), Box<dyn error::Error>> {
         &timing::duration_as_ms(&PohConfig::default().target_tick_duration).to_string();
     let default_ticks_per_slot = &clock::DEFAULT_TICKS_PER_SLOT.to_string();
     let default_slots_per_epoch = &EpochSchedule::default().slots_per_epoch.to_string();
+    let default_operating_mode = "production";
```

Comment:
"production" should be called "softlaunch" to match https://github.com/solana-labs/solana/blob/c6b108ef4fea939b87e7697848898bcfa5de9d9b/sdk/src/genesis_config.rs#L26